### ISSUE TITLE
Change input to InitMax, and other functionality changes

### ISSUE THREAD

**Issue by josuegaleas** [aahamed/SeniorDesign#8]
Modify InitMax method in [InitMax.java](https://github.com/aahamed/SeniorDesign/blob/josue/src/InitMax.java#L97) to take in the old distance matrix as a list rather than an array.

After running writing the rest of the cases for MainBody, I realized that the newly formed MST is not actually used. It's only used at the very end to graph the new MST, but Kai commented that part out. Really the most important output of InitMax is the new Distance matrix and the new list of HCS coordinates.

As a work-around, I made a few of your methods public and did this (first use can be found [here](https://github.com/aahamed/SeniorDesign/blob/josue/src/MainBody.java#L433)):

```
UV = InitMax.convertToHCS(XYr, XYc);
SUPER_TEMP = List_ops.ll2array(D2);
MORE_TEMP = InitMax.computeDistMatrix(UV);
EXTRA_TEMP = InitMax.mergeDistMat(SUPER_TEMP, MORE_TEMP);
D2 = List_ops.array2ll(EXTRA_TEMP);
```

SUPER_TEMP, MORE_TEMP, and EXTRA_TEMP are all two dimensional ordinary arrays, defined [here](https://github.com/aahamed/SeniorDesign/blob/josue/src/MainBody.java#L345).
- On the first line, I got your `convertToHCS` method and overwrote `UV`, which is list of coordinates type integer, defined [here](https://github.com/aahamed/SeniorDesign/blob/josue/src/MainBody.java#L288). This holds all the HCS coordinates.
- On the second line, I converted the old distance matrix `D2`, which is a list of lists of integers, defined [here](https://github.com/aahamed/SeniorDesign/blob/josue/src/MainBody.java#L326), to a 2D array.
- On the third line, I computed the new Distance matrix using your method `computeDistMatrix`, based off the newly formed `UV` list.
- On the fourth line, I used your `mergeDistMat` method to combine the two distance matrix into one.
- On the fifth line, I converted the combined distance matrix back to a List of lists, and overwrote `D2`

So I would say, maybe create a InitMaxOut class to have these three things?
Basically the new `UV` list, the new `D2` distance matrix, and also the new MST, for sake of consistency.
Something that starts like:

```
public class InitMaxOut
{
    private List<Coordinate<Integer> HCSList;
    private List<List<Integer>> DMatrix;
    private int[] parent;

    // Everything else
}
```

This however is just for making stuff looking pretty and more streamline, so maybe leave this as a last priority since the work-around does the same thing. For now we should focus more on getting the functionality down.

**Comment by aahamed** [comment]
I have added the enhancements requested. I have also added a simple test case in [test/TestInitMax.java](https://github.com/aahamed/SeniorDesign/blob/aadil/test/TestInitMax.java)

Check it out and see if it is ok. You can close the issue if you find the enhancements acceptable
